core/ui/Palettes/Preview/SideBar/Search.tid

```diff
@@ -0,0 +1,8 @@
+title: $:/core/ui/Palettes/Preview/Sidebar/Search
+tags: $:/tags/Preview/SideBar
+
+\whitespace trim
+<$colour.div class="tc-palette-preview-thumbnail-sidebar-search" styleName="background-color" styleColour="background">
```

Comment:
If parameters are named `style*` they immediately raise red flags in my mind. I associate those parameters with hardcoded STYLE tags in HTML. Hardcoded styles make colour customizing extremely hard. 

So I would like different widget parameter names if possible. 

I have to say I did not read the whole code yet. Only saw these parameters and go "triggered"

Reply:
>In any case, this custom widget is actually going to be removed soon

OK. After saving the post, I actually thought that they may be assigned to styles. -- If it is removed I'm relieved.

Reply:
Thanks @pmario